isthmus/src/test/java/io/substrait/isthmus/SimplePlansTest.java

```diff
@@ -1,59 +1,53 @@
 
   @Test
   public void aggFilter() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    s.execute("select sum(L_ORDERKEY) filter(WHERE L_ORDERKEY > 10) from lineitem ", creates);
+    assertProtoPlanRoundrip("select sum(L_ORDERKEY) filter(WHERE L_ORDERKEY > 10) from lineitem ");
   }
 
   @Test
   public void cd() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    // creates.forEach(System.out::println);
-    s.execute(
-        "select l_partkey, sum(distinct L_ORDERKEY) from lineitem group by l_partkey ", creates);
+    assertProtoPlanRoundrip(
+        "select l_partkey, sum(distinct L_ORDERKEY) from lineitem group by l_partkey ");
   }
 
   @Test
   public void filter() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    String[] values = asString("tpch/schema.sql").split(";");
-    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    // creates.forEach(System.out::println);
-    print(s.execute("select * from lineitem WHERE L_ORDERKEY > 10", creates));
+    assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY > 10");
   }
 
   @Test
-  public void joinWithMultiDDLInOneString() throws IOException, SqlParseException {
-    SqlToSubstrait s = new SqlToSubstrait();
-    List<String> ddlStmts = Lists.newArrayList(asString("tpch/schema.sql"));
-    print(
-        s.execute(
-            "select * from lineitem l, orders o WHERE o.o_orderkey = l.l_orderkey  and L_ORDERKEY > 10",
-            ddlStmts));
+  public void trailingSemicolon() throws IOException, SqlParseException {
+    assertProtoPlanRoundrip("select * from lineitem WHERE L_ORDERKEY > 10;");
   }
 
-  private void print(Plan plan) {
+  @Test
+  public void multiStatement() throws IOException, SqlParseException {
     try {
-      System.out.println(JsonFormat.printer().includingDefaultValueFields().print(plan));
-    } catch (InvalidProtocolBufferException e) {
-      throw new RuntimeException(e);
+      assertProtoPlanRoundrip(
+          "select l_orderkey from lineitem; select l_partkey from lineitem WHERE L_ORDERKEY > 20;");
+      fail();
+    } catch (UnsupportedOperationException e) {
+      assertTrue(e.getMessage().contains("SQL must contain only a single statement"));
     }
+    assertProtoPlanRoundrip(
+        "select l_orderkey from lineitem; select l_partkey from lineitem WHERE L_ORDERKEY > 20;",
+        new SqlToSubstrait(
+            new SqlToSubstrait.Options(SqlToSubstrait.StatementBatching.MULTI_STATEMENT)));
+  }
+
+  @Test
+  public void joinWithMultiDDLInOneString() throws IOException, SqlParseException {
```

Comment:
I don't understand why this is called joinWithMultiDDLInOneString.

Reply:
The name didn't change. It's just the diff that got a bit confused because the `multiStatement()` appears before it. I can reorder methods if you want.

Reply:
The name `joinWithMultiDDLInOneString` is a test I added for the scenario where one `--create` string contains multiple create_table DDLs.  It's not for multiple queries per input string.

Reply:
Reordered to help with diff